eureka-protempa-etl/src/main/java/edu/emory/cci/aiw/cvrg/eureka/etl/job/Task.java

```diff
@@ -164,8 +164,10 @@ public void run() {
 			this.etl.run(myJob, propDefArray, propIdsToShowArray, this.filter, this.updateData, this.prompts);
 			this.etl.close();
 			JobEvent completedJobEvent = new JobEvent();
+			Date jobFinishedDate = new Date();
+			myJob.setFinished(jobFinishedDate);
```

Comment:
Move this to after the job event creation, so the timestamps don't look strange (a job event occurring after the end timestamp of the job).